pkg/config/message_broker.go

```diff
@@ -25,6 +27,7 @@ func NewMessageBrokerConfig() *MessageBrokerConfig {
 }
 
 func (c *MessageBrokerConfig) AddFlags(fs *pflag.FlagSet) {
+	fs.BoolVar(&c.Disable, "disable-message-broker", c.Disable, "Disable MQTT message broker, default is false")
```

Comment:
how about to hidden this flag?
_ = fs.CommandLine.MarkHidden("disable-message-broker") 
I do not want to expose it externally.